model/Check/AbstractCheck.php

```diff
@@ -51,7 +51,7 @@ public function __construct(string $type, array $params = [])
         $this->details = $params[static::PARAM_DETAILS] ?? '';
     }
 
-    abstract public function __invoke($params): Report;
+    abstract public function __invoke($params = []): Report;
```

Comment:
1 broken test because of this change: taoSystemStatus\test\model\Check\SampleCheck::__invoke($params)